sass/hds/keycloak-overrides.scss

```diff
@@ -8,11 +8,18 @@ label.hds-checkbox__label {
 
 // Make sure that button links get the correct font size.
 a.hds-button {
-  font-size: $fontsize-body-default;
+  font-size: $fontsize-body-m;
 }
 
 // Make sure that patternfly doesn't add a margin to hds checkboxes
 // which break layout.
 input[type="checkbox"].hds-checkbox__input {
   margin-top: 0;
-}
\ No newline at end of file
+}
+// Fix ghost checkbox issue when checkbox/radio has focus
+// HDS changes input position to absolute and hides actual input
+// which causes weird issues with patternfly's (keycloak base css) outline declarations
```

Comment:
Is Patternfly actually needed for anything in this theme? As an example I checked the `helsinki/login/update-profile-confirm-dialog.ftl` page live and on that page almost all Patternfly's CSS properties are overridden by HDS and/or custom CSS from this theme. Would it be easier to just remove Patternfly altogether instead of trying to fight against it?

Reply:
https://helsinkisolutionoffice.atlassian.net/browse/HP-1499